frontend/public/locales/en/translation.json

```
@@ -574,14 +574,14 @@
       "tutorial": {
         "title1": "Welcome to Tournesol 🌻",
         "message1": {
-          "p10": "This tutorial will guide you step by step through a series of four comparisons.",
+          "p10": "This tutorial will guide you, step by step, through a series of four comparisons to enrich the project.",
```

I don't like "enrich" here. I'd propose "contribute to"

```suggestion
          "p10": "This tutorial will guide you, step by step, through a series of four comparisons to help you contribute to the Tournesol project",
```

---

Would recommend updating the French too

---

Fixed by https://github.com/tournesol-app/tournesol/pull/1110/commits/d30471e2fc49580aab2a884f98a82ec0da96f69c.